### PR TITLE
cloud-file-browser-ui-update

### PR DESCRIPTION
**Pull request by CollinEMac**
-Fixes breadcrumb caret issue
-Changes folder icons to a manila color and file icons to white.
-Display file location and last modified date columns.

JIRA link - https://divvyhq.atlassian.net/browse/DIV-7350

### CODE REVIEW

**Comment by nmallare** [comment]
@ganiserb or @letoss Can you please take a gander at this. Collin ran `npm i moment` and it made a bunch of node_module changes. Is this expected?

File: divvy/cloudfilebrowser.js [hunk 1]
```diff
@@ -1067,11 +1071,12 @@ var cloudFileBrowser = (function() {
                         tableHTML += '<li class="checkbox"></li>' +
                                      '<li class="filename">';
 
+                    var modifiedDate = moment(objItm.modifiedDate).format('MM/DD/YYYY');
+
                     tableHTML += objItm.name + '</li>' +
```
Comment (nmallare) [inline divvy/cloudfilebrowser.js:1072]: It might be working checking out what else might be available via the API with regards to metadata.

**Comment by letoss** [comment]
@nmallare I don't understand why node_modules have been versioned. This is incorrect.

**Comment by ganiserb** [comment]
I'm going to reply from email because my browser crashes every time I try
to load the PR diff...

Unversion node_modules :P

**Comment by CollinEMac** [comment]
@ganiserb and @letoss, I don't understand what you mean by "versioning" node_modules.

Are you guys saying that I can just update my package.json file without checking in changes to node_modules?

**Comment by ganiserb** [comment]
Exactly. Because when Divvy installs this dependency it will read
cloudelements' package_json file and install the dependencies in its file
and then it will install Divvy's dependencies. That's my understanding.

**Review by nmallare — APPROVED** [review]
GJ, mang.

**Pull request merged**